src/metrics_test.go

```diff
@@ -213,17 +213,17 @@ func TestGetBytes_InvalidDataKey(t *testing.T) {
 	}
 }
 
-var entity *integration.Entity
+var e *integration.Entity
```

Comment:
I would avoid having this global variable shared with different tests.

Reply:
I think this was added in order to avoid parsing the arguments multiple times. If I change this the sdk will panic